### PR TITLE
GitHub followers

### PR DESCRIPTION
**Pull request by coloniusrex**
Completes Story for GitHub Followers
- Creates Follower PORO
- Adds followers method to GithubService
- Adds user_followers GH API get request
- Adds followers section to user_show, adds repos section
- Updates to add followers variable to show controller
- Creates Tests for Followers section in User Dashboard Github

- [x] All tests passing?

### CODE REVIEW

**Comment by SMJ289** [comment]
We should try to run the Rubocop test before every pull request!

File: app/poros/github_response.rb [hunk 1]
```diff
@@ -10,4 +10,11 @@ def user_name
...
     json[:login]
   end
+
+  def followers
+    json = GithubService.new.user_followers
+    json.map do |follower_data|
+      Follower.new(follower_data)
+    end
+  end
 end
```
Comment (SMJ289) [inline app/poros/github_response.rb:20]: We can probably refactor the GithubService.new call into its own method

File: spec/features/user/user_dashboard_spec.rb [hunk 2]
```diff
@@ -18,9 +18,32 @@
...
+    end
+  end
 end
+
+
+# As a logged in user
+# When I visit /dashboard
+# Then I should see a section for "Github"
+# And under that section I should see another section titled "Followers"
+# And I should see list of all followers with their handles linking to their Github profile
```
Comment (SMJ289) [inline spec/features/user/user_dashboard_spec.rb:49]: We need to delete these commented out lines!

Comment (coloniusrex) [reply spec/features/user/user_dashboard_spec.rb]: Oh I deleted these

**Pull request merged**